pymonntorch/NetworkCore/NeuronGroup.py

```diff
@@ -22,7 +22,7 @@ class NeuronGroup(NetworkObject):
         id (torch.Tensor): The integer id of the population.
     """
 
-    def __init__(self, size, behavior, net, tag=None):
+    def __init__(self, size, net, behavior=None, tag=None):
```

Comment:
could we set a convention that net/network is always the first parameter

Reply:
There will be another PR fixing these issues plus type hints for arguments.